Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/ReflectometryILLPreprocessTest.py

```diff
@@ -119,6 +120,8 @@ def testDefaultRunFIGARO(self):
         self.assertAlmostEqual(numpy.rad2deg(two_theta_fg), 0., delta=0.1)
 
     def testTwoInputFiles(self):
+        ConfigService.Instance().setString("default.facility",   "ILL")
```

Comment:
Perhaps overriding setUp and expanding the already-present tearDown methods would be more elegant, the solution could look as follows:
```
    def setUp(self):
        self._facility = config['default.facility']
        self._instrument = config['default.instrument']

        config['default.facility'] = 'ILL'
        config['default.instrument'] = 'D17'

    def tearDown(self):
        if self._facility:
            config['default.facility'] = self._facility
        if self._instrument:
            config['default.instrument'] = self._instrument
        mtd.clear()
```

Reply:
Thanks for the suggestion. I have revised the section accordingly.